src/include/execution/sql/bloom_filter.h

```diff
@@ -0,0 +1,148 @@
+#pragma once
```

Comment:
@pmenon, is there anything about this bloom filter implementation that makes it only appropriate for the SQL execution engine?  If not, I think we should move it into `terrier::common` because there may be future situations where a bloom filter would be handy.

Reply:
Agreed, can be pulled into common, but it does rely on SIMD. There was code at the bottom of the header that implemented add and `Add()` and `Contains()` without SIMD that have to be brought back in, though.